packages/cli/src/commands/teams/list.ts

```diff
@@ -76,14 +77,22 @@ export default async function list(client: Client): Promise<number> {
   output.stopSpinner();
   client.stdout.write('\n'); // empty line
 
-  table(
-    ['', 'id', 'email / name'],
-    teamList.map(team => [team.current, team.value, team.name]),
-    [1, 5],
-    (str: string) => {
-      client.stdout.write(str);
-    }
+  const teamTable = table(
+    [
+      ['id', 'email / name'].map(str => gray(str)),
+      ...teamList.map(team => [team.value, team.name]),
+    ],
+    { hsep: 5 }
   );
+  client.stdout.write(
+    currentTeam
+      ? teamTable
+          .split('\n')
+          .map((line, i) => `${i > 0 ? teamList[i - 1].prefix : ' '} ${line}`)
+          .join('\n')
+      : teamTable
+  );
+  client.stdout.write('\n');
```

Comment:
```suggestion
  client.stderr.write('\n');
```
I _think_ we pretty much always want to write to `stderr`